src/partials/header.html

```diff
@@ -0,0 +1,47 @@
+<header class="header">
+  <div class="container header-top">
+    <nav class="header-nav">
+      <a href="./index.html" class="header-logo"
+        ><img
+          src="../images/logo.png"
+          width="80"
+          height="30"
+          alt="Логотип адский английский"
+      /></a>
+      <div class="header-nav-contact">
+        <ul class="header-list">
+          <li class="header-item">
+            <a href="#" class="header-link">Программы обучения</a>
+          </li>
+          <li class="header-item">
+            <a href="#" class="header-link">О нас</a>
+          </li>
+          <li class="header-item">
+            <a href="#" class="header-link">Варианты обучения</a>
+          </li>
+          <li class="header-item">
+            <a href="#" class="header-link">Контакты</a>
+          </li>
+        </ul>
+        <button type="button" class="header-btn-consult">
+          Консультация по телефону
+        </button>
+        <button
+          type="button"
+          class="btn-burger"
+          aria-expanded="false"
+          data-modal-open
+        >
+          <!-- <svg
+            class="header-icon-menu"
+            width="24"
+            height="12"
+            aria-label="Переклчатель мобильного меню"
+          >
+            <use class="header-icon-menu" xlink:href="../images/sprite.svg#menu"></use>
```

Comment:
Нужно заменить 2 точки на 1, как сделано на примере снизу:

```suggestion
            <use class="header-icon-menu" xlink:href="./images/sprite.svg#menu"></use>
```